deluge/plugins/AutoAdd/deluge/plugins/autoadd/core.py

```diff
@@ -253,7 +253,7 @@ def update_watchdir(self, watchdir_id):
                 continue
             else:
                 ext = os.path.splitext(filename)[1]
-                if ext == ".torrent":
+                if ext == ".torrent" or ".TORRENT":
```

Comment:
`ext == ".torrent" or ".TORRENT"` will resolve to True when ext contains ".torrent". If it does not contain ".torrent", it will resolve to ".TORRENT". This means the if test expression will always be true, and `magnet` set to `False`